modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_azure/installing-azure-government-region.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+// * installing/installing_azure/installing-azure-private.adoc
+// * installing/installing_azure/installing-azure-vnet.adoc
+
+:_content-type: PROCEDURE
+[id="finalizing-encryption_{context}"]
+= Finalizing user-managed encryption after installation
+If you installed {product-title} using a user-managed encryption key, you can complete the installation by updating the default storage class and granting `Owner` permissions to the Azure cluster resource group.
+
+.Procedure
+
+. Update the annotations on the default storage class to remove it as the default. The storageClass that uses the encrypted storage will be the default storageClass instead.
++
+[source,terminal]
+----
+$ oc patch storageclass managed-premium -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+----
++
+. Obtain the identity of the cluster resource group used by the installer:
+.. If you specified an existing resource group in `install-config.yaml`, obtain its Azure identity:
++
+[source,terminal]
+----
+$ az identity list --resource-group "<existing_resource_group>"
+----
++
+.. If you did not specify a existing resource group in `install-config.yaml`, locate the resource group that the installer created, then obtain its Azure identity:
++
+[source,terminal]
+----
+$ az group list
+----
++
+[source,terminal]
+----
+$ az identity list --resource-group "<installer_created_resource_group>"
+----
+. Grant the `Owner` role assignment to the cluster resource group so that it can write to the Disk Encryption Set:
++
+[source,terminal]
+----
+$ az role assignment create --role Owner --assignee "<resource_group_identity>"
```

Comment:
"Owner" isn't a requirement. A role that has write permissions to the disk encryption set will be necessary however.